### PR TITLE
Ocurrent benchmarking

### PR DESCRIPTION
**Pull request by rgrinberg**
Add a simple benchmarking job that creates a small workspace and measures how
long is a clean/zero build.

### CODE REVIEW

File: dune-project [hunk 1]
```diff
@@ -156,3 +156,12 @@ understood by dune language."))
...
  (description "This library offers no backwards compatibility guarantees. Use at your own risk."))
+
+(package
+ (name dune-bench)
+ (synopsis "Dune's unstable standard library")
+ (depends
+  (ocaml (>= 4.08.0))
+  pp
+  csexp)
+ (description "This library offers no backwards compatibility guarantees. Use at your own risk."))
```
Comment (Khady) [inline dune-project:167]: description and synopsis aren't correct

Comment (rgrinberg) [reply dune-project]: Fixed.

File: bench/inline/dune [hunk 2]
```diff
@@ -0,0 +1,13 @@
+(executable
```
Comment (jeremiedimino) [inline bench/inline/dune:1]: I'd call this directory "micro-benchmarks" rather than "inline".

**Review by jeremiedimino — APPROVED** [review]
Looks good! Eager to try this out :)

File: bench/inline/dune (hunk 2 above)
Comment (rgrinberg) [reply bench/inline/dune:1]: I settled for `micro`. It's already in the `bench` dir, so it's clear it's a benchmark.

**Pull request merged**